src/config.rs

```diff
@@ -141,4 +142,40 @@ impl ParserConfig {
         self.transform_block = Some(Rc::new(callback));
         self
     }
+
+    /// Allows unmatched tag pairs where the element opens with a block and
+    /// the close tag matches a specified wildcard. For example
+    ///
+    /// ```rust
+    /// use quote::quote;
+    /// use rstml::{Parser, ParserConfig, node::{Node, NodeElement}};
+    /// use syn::{Expr, ExprRange, RangeLimits, Stmt};
+    ///
+    /// let config = ParserConfig::new().block_element_close_wildcard(|close_tag| {
+    ///     // Construct a sample of what a wildcard block element close should be
+    ///     let sample_close = quote! { <{}></_> };
+    ///     let parsed = Parser::new(ParserConfig::new().block_element_close_wildcard(|_| true))
```

Comment:
Could you add example that will check that wildcard is Ident("_").

As a shortcut you can also add new method to `ParserConfig`: `use_default_close_wildcard(mut self)` that will hide closure usage for end-user.